apps/src/javalab/JavalabEditor.jsx

```diff
@@ -85,6 +93,51 @@ class JavalabEditor extends React.Component {
     });
   }
 
+  makeListeners(key) {
+    return {
+      onContextMenu: e => {
```

Comment:
for my information: how does onContextMenu work?

Reply:
It's the same as on right click, it just overrides the existing menu that shows up and displays our menu instead